### PR TITLE
adding NetBox icon

### PR DESCRIPTION
**Pull request by jorgeamadosoria**
netbox

issue: #7058

### CODE REVIEW

**Review by sachinraja — CHANGES_REQUESTED** [review]
Differences along the edges on the right side:
image

**Comment by jorgeamadosoria** [comment]
I've removed the gaps. The icon is now exactly the same one as it is in the svg source.

**Review by dirien — CHANGES_REQUESTED** [review]
Hi @jorgeamadosoria 

I see some differences:

image


I use https://netbox.readthedocs.io/en/stable/netbox_logo.svg as base

**Comment by jorgeamadosoria** [comment]
I updated the icon with precision 5. I don't see differences. Please check @dirien

**Comment by sachinraja** [comment]
Still seeing differences:
image

Not sure what's going on

**Comment by jorgeamadosoria** [comment]
> Still seeing differences: image
> 
> Not sure what's going on

what are you using for comparison?

**Comment by sachinraja** [comment]
I used https://netbox.readthedocs.io/en/stable/netbox_logo.svg

File: _data/simple-icons.json [hunk 1]
```diff
@@ -6566,6 +6566,12 @@
             "source": "http://www.netapp.com/",
             "guidelines": "https://www.netapp.com/company/legal/trademark-guidelines/"
         },
+        {
+            "title": "NetBox",
+            "hex": "1685FC",
+            "source": "https://netbox.readthedocs.io/en/stable/development/style-guide/",
```
Comment (adamrusted) [inline _data/simple-icons.json:6572]:
```suggestion
            "source": "https://netbox.readthedocs.io",
```

The logo SVG itself can be found on the homepage.

File: icons/netbox.svg
Comment (adamrusted) [inline icons/netbox.svg:1]: This might be better with gaps to delineate the different colours?

**Review by adamrusted — CHANGES_REQUESTED** [review]

**Comment by jorgeamadosoria** [comment]
Closing as I have regrettably withdrawn from this project due to personal circumstances.

**Pull request closed without merging**